web-console/src/utils/general.tsx

```diff
@@ -88,9 +88,10 @@ function getNeedleAndMode(input: string): NeedleAndMode {
 }
 
 export function booleanCustomTableFilter(filter: Filter, value: any): boolean {
-  if (value === undefined) {
+  if (value === undefined ) {
```

Comment:
Why?

Reply:
because only running tasks have locations so value was null and would error, so I am just automatically excluding anything with out a location.

Reply:
but why did you add the space between `undefined` and `)`